features/alarm-change-notifier/main-module/src/test/java/org/opennms/plugins/dbnotifier/test/manual/SimpleJsonTest.java

```diff
@@ -0,0 +1,257 @@
+		String testString2="{\"alarmid\":285,\"eventuei\":\"uei.opennms.org/application/generic/piAlarmRaise\",\"nodeid\":null,\"ipaddr\":\"127.0.0.1\",\"serviceid\":null,\"reductionkey\":\"uei.opennms.org/application/generic/piAlarmRaise:0:127.0.0.1:1\",\"alarmtype\":1,\"counter\":2,\"severity\":5,\"lasteventid\":4160,\"firsteventtime\":\"2016-07-04 16:19:14+01\",\"lasteventtime\":\"2016-07-04 16:24:04+01\",\"firstautomationtime\":null,\"lastautomationtime\":null,\"description\":\"Generic Raspberry Pi Alarm Raise 1\",\"logmsg\":\"Generic Raspberry Pi Alarm Raise 1\",\"operinstruct\":\"\",\"tticketid\":null,\"tticketstate\":null,\"mouseovertext\":null,\"suppresseduntil\":\"2016-07-04 16:19:14+01\",\"suppresseduser\":null,\"suppressedtime\":\"2016-07-04 16:19:14+01\",\"alarmackuser\":\"admin\",\"alarmacktime\":\"2016-07-05 13:38:51.753+01\",\"managedobjectinstance\":null,\"managedobjecttype\":null,\"applicationdn\":null,\"ossprimarykey\":null,\"x733alarmtype\":null,\"x733probablecause\":0,\"qosalarmstate\":null,\"clearkey\":null,\"ifindex\":null,\"eventparms\":\"PiIoId=1(string,text)\",\"stickymemo\":null,\"systemid\":\"00000000-0000-0000-0000-000000000000\"}";
+
+		// "suppressedtime":"2016-07-04 16:19:14+01","systemid":"00000000-0000-0000-0000-000000000000","suppresseduntil":"2016-07-04 16:19:14+01","description":"Generic Raspberry Pi Alarm Raise 1","mouseovertext":null,"x733probablecause":0,"lasteventid":4184,"lasteventtime":"2016-07-05 12:00:02+01","managedobjectinstance":null,"alarmacktime":"2016-07-05 13:38:51.753+01","qosalarmstate":null,"ipaddr":"127.0.0.1","alarmackuser":"admin","nodeid":null,"firsteventtime":"2016-07-04 16:19:14+01","severity":5,"ifindex":null,"alarmtype":1,"x733alarmtype":null,"logmsg":"Generic Raspberry Pi Alarm Raise 1","tticketid":null,"firstautomationtime":null,"clearkey":null,"managedobjecttype":null,"eventuei":"uei.opennms.org\/application\/generic\/piAlarmRaise","counter":3,"applicationdn":null,"operinstruct":"","ossprimarykey":null,"stickymemo":null,"tticketstate":null,"alarmid":285,"serviceid":null,"reductionkey":"uei.opennms.org\/application\/generic\/piAlarmRaise:0:127.0.0.1:1","suppresseduser":null,"lastautomationtime":null,"eventparms":"PiIoId=1(string,text)"
+		//String testString = "[{},{}]";
+
+		String testString = "["+testString1+","+testString2+"]";
```

Comment:
This is simply a manual developer test to ensure we can parse the database JSON notification content into an alarm.